content/statefulset/storageclass.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Define Storageclass"
+
+date: 2018-08-07T08:30:11-07:00
+weight: 5
+---
+#### Introduction
+[Dynamic Volume Provisioning](http://localhost:1313/statefulset/storageclass/) allows storage volumes to be created on-demand. [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) should be pre-created to define which provisoner should be used and what parameters should be passed when dynamic provisioning is invoked.
+(See parameters for [AWS EBS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs))
+#### Define Storage Class
+Copy/Paste the following commands into your Cloud9 Terminal. 
+```
+mkdir ~/environment/templates
+cd ~/environment/templates
+wget https://eksworkshop.com/statefulset/storageclass.files/mysql-storageclass.yml
+```
+Check the configuration of mysql-storageclass.yml file by following command.
+```
+cat ~/environment/templates/mysql-storageclass.yml
+```
+You can see provisioner is kubernetes.io/aws-ebs and parameters for EBS.
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: mysql-gp2
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+  zones: us-west-2a, us-west-2b, us-west-2c
```

Comment:
I would remove 'Zones' altogether. This will remove stickiness with respect to regions and AZ's. I was using us-east-1 and the audience may not have one of the AZ's defined in there account.

Reply:
AZ parameter is deleted at storageclass